app/src/main/java/com/example/plantsapp/presentation/ui/notification/NotificationWorker.kt

```diff
@@ -7,23 +7,23 @@ import androidx.work.WorkerParameters
 import com.example.plantsapp.domain.repository.TasksRepository
 import dagger.assisted.Assisted
 import dagger.assisted.AssistedInject
-import kotlinx.coroutines.flow.first
 import java.util.Date
 
 @HiltWorker
 class NotificationWorker @AssistedInject constructor(
     @Assisted private val context: Context,
     @Assisted workerParams: WorkerParameters,
     private val repository: TasksRepository,
-    private val notificationManager: TaskNotificationManager
+    // private val notificationManager: TaskNotificationManager
 ) : CoroutineWorker(context, workerParams) {
 
+    // TODO show notifications only for uncompleted tasks
     override suspend fun doWork(): Result {
         val plantsWithTasks =
-            repository.getPlantsWithTasksForDate(Date()).first()
+            repository.getPlantsWithTasksForDate(Date())
 
         plantsWithTasks.forEach { (plant, tasks) ->
-            notificationManager.showTaskNotifications(plant, tasks)
+             // notificationManager.showTaskNotifications(plant, tasks)
```

Comment:
Uncomment and just map + add todo here , instead of breaking this think completely